nss/Survey_NSS/02_Table of Means NSS 70.do

```diff
@@ -67,35 +68,42 @@ bysort urban: sum legal_own [aw = hhwgt],de
 ****************************************************************************
 use "${r_input}\Visit 1_Block 14",clear
 
-*debt
-gen debt = b14_q17
+*debt -- paper says liabilities on date of survey. pp16: "Panel A of Figure 2 reports the average allocation of liabilities across all households that carry a positive amount of debt at the date of the survey."
+gen debt = b14_q17 // 17 is on 30.06.2012
```

Comment:
pp13: "Turning to liabilities, all types of personal liabilities are captured as amounts outstanding in June 2012"
b14_q17 is used in estimating the liability amount. 

pp16: "Panel A of Figure 2 reports the average allocation of liabilities across all households that carry a positive amount of debt at the date of the survey."
b14_q16 is used as a filter, will reflect this on the code.